service/initialise.go

```diff
@@ -117,8 +118,8 @@ func (e *ExternalServiceList) GetS3Clients(cfg *config.Config) (s3Private, s3Pub
 }
 
 // GetS3Client returns S3 clients private and public. They share the same AWS session.
```

Comment:
```suggestion
// GetS3Client returns S3 clients private and public. They share the same AWS config.
```
This appears in a couple comments elsewhere in the codebase